pages/curriculum.tsx

```diff
@@ -69,7 +71,7 @@ export const Curriculum: React.FC<GetAppProps> = ({ data }) => {
         currentState={lessonState}
         reviewUrl={`/review/${id}`}
         challengesUrl={`/curriculum/${id}`}
-        docUrl={lesson?.docUrl || ''}
+        docUrl={lesson.docUrl || ''}
```

Comment:
Shouldn't this be `_.get(lesson, 'docUrl', '')` or does the GraphQL definition already guarantee that it exists?

Reply:
I thought since it's got a fallback `|| ''`. Wasn't completely sure about it.

Reply:
I believe @hwong0305 is right. If `lesson` is undefined, then `lesson.docUrl` would cause a TypeError

Reply:
Using `_.get(lesson, 'docUrl', '')` creates a type error saying `type Maybe<String> is not assignable to type string`. How can I fix that?
If lesson is undefined an error would be thrown earlier saying `<h1>Bad Data</h1>`

Reply:
Try `_.get(lesson, 'docUrl', '') as string || ''`